peer/pb/peer_record.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package peer.pb;
+
+message PeerRecord {
+    message AddressInfo {
+        bytes multiaddr = 1;
+    }
+
+    bytes peerId = 1;
```

Comment:
(bikeshed alert) Pretty please, can we use Protobuf style conventions and name fields with snake case? https://developers.google.com/protocol-buffers/docs/style#message-and-field-names

libp2p has upper camel, lower camel, and snake casing across the codebase, and the correct one is the latter :-) Fortunately, fixing the casing in other protobufs should not break anything.

Reply:
@yusefnapora you may have missed this comment?